src/test/java/hudson/plugins/sonar/BaseTest.java

```diff
@@ -108,17 +110,20 @@ protected void setBuildResult(Project<?, ?> project, Result result) throws Excep
   public void testPassword() throws Exception {
     configureDefaultSonarRunner(false);
     configureSecuredSonar();
+
     FreeStyleProject project = setupFreeStyleProjectWithSonarRunner();
     Run<?, ?> build = build(project);
 
     assertLogContains("sonar-runner", build);
-    assertLogDoesntContains("sonar.login", build);
+    assertLogContains("-Dsonar.host.url=http://example.org:9999/sonar ********", build); // check masked token is in logs
```

Comment:
This test previously passed when the token wasn't even being added (I wasn't adding the credential properly)
This change asserts that some masked thing goes in, in the expected place

Reply:
I've added an extra assertion looking for the secret to be extra sure it doesn't go in the log